uims_api/session.py

```diff
@@ -79,6 +79,14 @@ def _get_attendance(self):
         # These cookies contain encoded information about the current logged in UID whose
         # attendance information is to be fetched
         response = requests.get(attendance_url, cookies=self.cookies)
+        if(response.text.find(ERROR_HEAD)):
+            raise UimsFailureError('UIMS internal error occured')
```

Comment:
Is it necessary to search for this string in the html source? If the request to the attendance URL returns an error response code (code 500 maybe for internal error?), we could probably check for this error code instead of searching for the string.

If it indeed returns an error code, we should also update the message in the exception raised to include this error code as it seems more informative.

Reply:
Yeah, i am looking for this now ...

Reply:
Checked, status is 200 !
String searching is required

Reply:
Ok. Let's keep this as it is then and add a comment here about the response still being 200 so the next doesn't guy doesn't waste their time thinking about this!

Reply:
Yeah thats true, adding ..